Kitodo/src/main/java/org/kitodo/services/image/GenerationMode.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.services.image;
```

Comment:
Please add enums to org.kitodo.enums

Reply:
I did, even though I find it awful, because I had to declare quite a few methods `public` that should not be. The enum is very closely linked to the image generator, is only used internally in it, and has no other purpose.